src/skills-builder/skills-builder-modal/view-results/RelatedSkillsSelectableBoxSet.jsx

```diff
@@ -23,7 +26,7 @@ const RelatedSkillsSelectableBoxSet = ({ jobSkillsList, selectedJobTitle, onChan
   return (
     <SelectableBox.Set
       name="selected job title"
-      type="radio"
+      type={allowMultipleCareerInterests ? 'radio' : null}
```

Comment:
This seems like potentially an unnecessary check. The component is a `SelectableBoxSet` so it should be expecting multiple `careerInterests`. If there is an instance where this component is rendered when it should be a single box, then there is probably an issue with the parent component that renders it. Also, I don't think we should be passing `null` to the `type` prop as it is expecting it to be "radio" or "checkbox"

Reply:
I cleared all the experimental stuff out